components/gitpod-protocol/src/workspace-instance.ts

```diff
@@ -91,14 +91,6 @@ export interface WorkspaceInstanceStatus {
     // exposedPorts is the list of currently exposed ports
     exposedPorts?: WorkspaceInstancePort[];
 
-    /**
-     * repo contains information about the Git working copy inside the workspace
-     * @deprecated use WorkspaceInstance.gitStatus instead if supervisor_live_git_status feature flag is enabled
-     *
-     * TODO(ak) remove after migration to live git status
-     */
-    repo?: WorkspaceInstanceRepoStatus;
```

Comment:
@geropl is it safe? if there are old workspaces in DB with such property in status, it will be omitted, but otherwise work?

Reply:
Yes, because typeorm effectively uses JSON.parse and passes us the result. :+1: